debug/index.html

```diff
@@ -27,33 +27,33 @@
 });
 
 map.once('load', () => {
-    map.setLayoutProperty('building-number-label', 'symbol-clip',
-    ["case",
-        ["<", ["pitch"], 60], false,
-        ["all", [">=", ["pitch"], 60], ["<", ["distance-from-center"], 2]], false,
-        true
-    ]
-    );
-
-    map.setLayoutProperty('poi-label', 'symbol-clip',
-        ["case",
-            ["<", ["pitch"], 60], false,
-            ["all", [">=", ["pitch"], 60], ["<", ["distance-from-center"], 4]], false,
-            true
-        ]
-    );
-
-    map.setLayoutProperty('transit-label', 'symbol-clip',
-    ["case",
-        ["<", ["pitch"], 60], false,
-        ["all", [">=", ["pitch"], 60], ["<", ["distance-from-center"], 3]], false,
-        true
-    ]
-    );
-
-    map.once('idle', () => {
-        map.panTo([-122.4199724, 37.7687162], {duration: 50000});
-    });
+    // map.setFilter('building-number-label',
```

Comment:
Should there be a separate debug file for testing the commented out code instead of left commented out on the debug/index.html file?

Reply:
Could be nice as a separate page with mapbox-gl-compare so you can see the difference this filter has.

Reply:
I chose to get rid of this instead, I'll be adding a separate debug page in a followup PR that also adds render tests. Render-tests and the debug page will use the same geojson data so it should make debugging easier.

Not gonna go for gl-compare for this because it makes debugging harder because you can't easily tell which instance of the map is triggering a breakpoint.